Automattic/wp-calypso
# Theme Showcase: Improve upgrade banners on LiTS

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4628

## Proposed Changes

* Simplified the conditions checked to show the upgrade banner.
* Extracted building the banner to a separate method (separation of concerns)
* Now that we have extracted the banner building we can extend it in the future if needed without changing the main component.

## Testing Instructions



* Open the Calypso live link
* Go to the LiTS and look for the `Design your own banner`
* You should only see the banner on simple sites that are on a Premium plan.
* Test on Simple, Atomic and Jetpack site, with monthly, yearly and bi-yearly plans.

No before and after images provided since there are no visual changes in the banners.

## Pre-merge Checklist
